### ISSUE TITLE
Comma (,) token included model

### ISSUE THREAD

**Issue by sime** [angular-ui/ui-select#757]
When using a comma (,) as a tagging token, the comma reminds appended to the model. Ideally the token should be removed from the model.

http://plnkr.co/edit/NbNTG9yR7PJquhtIJdJU?p=preview

![screen shot 2015-03-13 at 16 49 59](https://cloud.githubusercontent.com/assets/216917/6641896/098a9df0-c9a1-11e4-94b9-1018d2c59d9a.png)

**Comment by dmitryevseev** [comment]
+1

**Comment by cyrilf** [comment]
:+1: that's the behavior I was expecting too

**Comment by cyrilf** [comment]
Just for your information, I figure out that adding the attribute ` tagging-label=""` to your `ui-select` directive will solve this bug. [See this Plnkr](http://plnkr.co/edit/w3d11j9l2pg1RhqtWuHU?p=preview)
I specified an empty string as I don't want any value to be displayed, but as in the Plunkr, you can specify one.

Hope it helps!

(Still a bug thought!)

**Comment by sime** [comment]
The workaround is valid, though you need the following CSS to mimic the desired functionality.

```css
.ui-select-choices {
  display: none;
}
```

**Comment by roko-p** [comment]
Yeah, so when you have the `tagging-label="false"` (when you don't want items appearing in the dropdown), you get the comma appended to the model.

One workaround is the css hack that @sime suggested above, another one is to set `tagging=tagTransform` where 

    $scope.tagTransform = function (newTag) {
            // remove the comma at the end if it exists
            if(newTag[newTag.length - 1] === ',') {
                newTag = newTag.slice(0, -1);
            }
            return newTag;
        };

...both ugly!

+1!

**Comment by roko-p** [comment]
I ended up creating this directive which I use instead of `tagging-label="false"` (so I just put `my-hide-choices-dropdown` where `tagging-label="false"` would go)

	angular.module('myApp')
	    .directive('myHideChoicesDropdown', function() {
	        return {
	            link: function(scope, element) {
	                element.find('.ui-select-choices').hide();
	            }
	        };
	    });

Thanks @sime !

**Comment by cesar-oyarzun-m** [comment]
hey @roko-p I try your solution and also the solution with css but both give me erros, here is the error

```TypeError: Cannot assign to read only property '_uiSelectChoiceDisabled' of label undefined```
